src/test/scala/org/hammerlab/guacamole/util/TestUtil.scala

```diff
@@ -231,11 +232,30 @@ object TestUtil extends Matchers {
       Pileup(normalReads, contig, locus, reference.getContig(contig)))
   }
 
-  def loadPileup(sc: SparkContext, filename: String, reference: ReferenceBroadcast, locus: Long = 0, contig: Option[String] = None): Pileup = {
-    val records = TestUtil.loadReads(sc, filename, reference = reference).mappedReads
+  def loadPileup(sc: SparkContext,
+                 filename: String,
+                 reference: ReferenceBroadcast,
+                 locus: Long = 0,
+                 contigOpt: Option[String] = None): Pileup = {
```

Comment:
Yeah I might be being overly pedantic here. I've used the convention
`maybeContig` elsewhere, but really anything here you think is best works
for me

On Fri, Apr 1, 2016 at 12:32 PM, Ryan Williams <notifications@github.com>
wrote:

> In src/test/scala/org/hammerlab/guacamole/util/TestUtil.scala
> <https://github.com/hammerlab/guacamole/pull/414#discussion_r58231292>:
>
> > @@ -231,11 +232,30 @@ object TestUtil extends Matchers {
> >        Pileup(normalReads, contig, locus, reference.getContig(contig)))
> >    }
> >
> > -  def loadPileup(sc: SparkContext, filename: String, reference: ReferenceBroadcast, locus: Long = 0, contig: Option[String] = None): Pileup = {
> > -    val records = TestUtil.loadReads(sc, filename, reference = reference).mappedReads
> > +  def loadPileup(sc: SparkContext,
> > +                 filename: String,
> > +                 reference: ReferenceBroadcast,
> > +                 locus: Long = 0,
> > +                 contigOpt: Option[String] = None): Pileup = {
>
> Also Options and the naming patterns here are so basic that I don't think
> this is a slippery slope to other inscrutable/abbreviated variable names,
> if that is your concern.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/hammerlab/guacamole/pull/414/files/91cec319662d7e270b069a0e86a2564a10f24ae7#r58231292>
>